src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/InlineData.cs

```diff
@@ -97,15 +97,16 @@ public static string DesignerWithContents (string projectName, string className,
 			}}
 		}}" + Environment.NewLine, data);
 			}
-			return Designer.Replace ("@className@", className)
+			return Designer.Replace ("@modifier@", modifier)
+				.Replace ("@className@", className)
 				.Replace ("@projectName@", projectName)
 				.Replace ("@content@", content.ToString ());
 		}
 
 		public static void AddCultureResourceDesignerToProject (IShortFormProject proj, string projectName, string className, params string[] dataNames)
 		{
 			proj.OtherBuildItems.Add (new BuildItem.Source ($"{className}.Designer.cs") {
-				TextContent = () => InlineData.DesignerWithContents (projectName, className, dataNames)
+				TextContent = () => InlineData.DesignerWithContents (projectName, className, "internal", dataNames)
```

Comment:
This line is hardcoded `className: "internal"`, is this for a specific case?

Reply:
no its just a default value to use

Reply:
I misread, didn't see this is test data.